bundle/compliance/cis_gcp/rules/cis_1_11/rule.rego

```diff
@@ -0,0 +1,35 @@
+package compliance.cis_gcp.rules.cis_1_11
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import data.compliance.policy.gcp.iam.ensure_admin_without_multiple_roles as audit
+import future.keywords.if
+
+default admin_has_other_role = false
+
+# check admin is not also cryptoKeyEncrypter
+admin_has_other_role if {
+	audit.admin_has_multiple_roles("roles/cloudkms.admin", "roles/cloudkms.cryptoKeyEncrypter")
+}
+
+# check admin is not also cryptoKeyDecrypter
+admin_has_other_role if {
+	audit.admin_has_multiple_roles("roles/cloudkms.admin", "roles/cloudkms.cryptoKeyDecrypter")
+}
+
+# check admin is not also cryptoKeyEncrypterDecrypter
+admin_has_other_role if {
+	audit.admin_has_multiple_roles("roles/cloudkms.admin", "roles/cloudkms.cryptoKeyEncrypterDecrypter")
+}
```

Comment:
super nit: the repo is pretty inconsistent with it, but I like keeping the finding part at the top and the helper functions bellow